stm32data/__main__.py

```diff
@@ -839,7 +839,13 @@ def parse_chips():
             for p in peris:
                 chs = []
                 for dma in chip_dmas:
-                    if peri_chs := dma_channels[dma]['peripherals'].get(p['name']):
+                    peri_chs = dma_channels[dma]['peripherals'].get(p['name'])
+
+                    # DAC1 is sometimes interchanged with DAC
```

Comment:
This feels really sketchy, but I'm not sure if there is a better place to fix this.

It can't be done in `parse_dma()`, because you can't know if chip is using `DAC` or `DAC1` without multiple ifs for specific chips.